### DISCUSSION TITLE
How do I direct message a member?

### DISCUSSION THREAD

**Discussion by cgcmgr** [GerryDazoo/Slinger#286] [Q&A]
Hi everyone,

This is my first new discussion.  I see a lot of members asking how to set up Slinger on a Windows PC.  I have written up pretty extensive tutorial with pictures that I'd like to post.  However, I'd like to send it to Gerry first so that he can read it over and give it the Ok before I do so.  Maybe even offer some changes or corrections.  After all, this is his project and I don't want to step on anyone's toes.  

Is there a way I can get this to Gerry without attaching it here first?

Thanks,
Chris

**Comment by GerryDazoo** [comment]
Someone over on the facebook windows group https://www.facebook.com/groups/377795270002137/permalink/800605857721074
wrote this.

[Step-by-step.pdf](https://github.com/GerryDazoo/Slinger/files/9997979/Step-by-step.pdf)

DM me over on reddit.  https://www.reddit.com/r/slingbox   https://www.reddit.com/user/Accomplished-Sign428/
